### PR TITLE
bug fixed

### PR DESCRIPTION
**Pull request by 1killerbehindyou1**
The current value set with the slider is used to render the image.

### CODE REVIEW

File: BasicGUI/QMLSources/main.qml [hunk 1]
```diff
@@ -117,6 +117,50 @@ ApplicationWindow
...
+       //modality: Qt: : WindowModality
+       //nameFilters: list<string>
+       //selectExisting: bool
+       //selectFolder: bool
+       //selectMultiple: bool
+       //selectedNameFilter: string
+       //shortcuts: Object
+       //sidebarVisible: bool
+       //title: string
+       //visible: bool
```
Comment (Nutek) [inline BasicGUI/QMLSources/main.qml:135]: Why commented?

File: BasicGUI/QMLSources/main.qml [hunk 2]
```diff
@@ -117,6 +117,50 @@ ApplicationWindow
...
+    FileDialog 
+    {
+        id: fileSaveDialog
+        title: "Save file"
+        folder: StandardPaths.writableLocation(StandardPaths.DocumentsLocation)
+        //nameFilters: [ "Image files (*.jpg *.png)", "All files (*)" ]
+        //selectMultiple: false
+        //currentFile: "file:///"
+        //folder: StandardPaths.writableLocation(StandardPaths.DocumentsLocation)
+        //fileMode: FileDialog.SaveFile | FileDialog.OpenFile
```
Comment (Nutek) [inline BasicGUI/QMLSources/main.qml:146]: Comment?

**Review by Nutek — CHANGES_REQUESTED** [review]

File: BasicGUI/QMLSources/main.qml (hunk 2 above)
Comment (1killerbehindyou1) [reply BasicGUI/QMLSources/main.qml]: removed

File: BasicGUI/QMLSources/main.qml (hunk 1 above)
Comment (1killerbehindyou1) [reply BasicGUI/QMLSources/main.qml]: removed

**Review by Nutek — APPROVED** [review]

**Pull request merged**